app/(main)/components/SyncUpdateButton/SyncUpdateButton.tsx

```
@@ -0,0 +1,15 @@
+import clsx from 'clsx';
+import { format } from 'date-fns/format';
+import { Icon } from '@/components/ui';
+import styles from './SyncUpdateButton.module.scss';
+
+export default function SyncUpdateButton({ lastSyncedAt, onClick }: { lastSyncedAt: string; onClick: () => void }) {
+  const formattedLastSyncedAt = lastSyncedAt ? format(new Date(lastSyncedAt), 'yy년 M월 d일 HH:mm') : '';
+
+  return (
+    <button className={clsx(styles.container, styles.text, '--body-sm-medium')} onClick={onClick}>
+      {formattedLastSyncedAt} 업데이트
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)